Authenticator/Source/TokenFormViewController.swift

```diff
@@ -195,8 +199,8 @@ class TokenFormViewController: UITableViewController {
 extension TokenFormViewController {
     // MARK: Bar Button View Model
 
-    private func barButtomItemForViewModel(viewModel: BarButtonViewModel, target: AnyObject?, action: Selector) -> UIBarButtonItem {
-        func systemItemForStyle(style: BarButtonViewModel.Style) -> UIBarButtonSystemItem {
+    private func barButtomItemForViewModel(viewModel: BarButtonViewModel<Form.Action>, target: AnyObject?, action: Selector) -> UIBarButtonItem {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 145 characters (line_length)